internal/docker/docker.go

```diff
@@ -217,7 +271,18 @@ func getAppName(imageName string) string {
 func getBinaryName(imageName string) string {
 	imageName = strings.TrimPrefix(imageName, fmt.Sprintf("%s/", imageBase()))
 	imageName = strings.Split(imageName, ":")[0]
-	return strings.Split(imageName, "/")[1]
+
+	// checks if there is a multiple projects or not.
+	// minside:v2025.09.18211159,ocreg.invalid/coopnorge/minside:latest
+	// helloworld/helloworld/cmd:v2025.09.18211159,ocreg.invalid/coopnorge/helloworld:latest
+	// helloworld/goodbyeworld/cmd:v2025.09.18211159,ocreg.invalid/coopnorge/goodbyeworld:latest
+	nameParts := strings.Split(imageName, "/")
+
+	if len(nameParts) > 1 {
+		return nameParts[1]
+	}
+
+	return ""
```

Comment:
I dont really understand here why this is necessary. This seems to come from the situation where the OCI image has no base (so no ocreg.invalid/coopnorge or docker.io etc etc). Is this a situation we want to support? A return of "" seems a bit strange when running 'getBinaryName'